libs/designsystem/item/src/item.component.html

```diff
@@ -2,6 +2,7 @@
   lines="none"
   [disabled]="disabled"
   [button]="_renderAsButton ? true : null"
+  [href]="_renderAsLink ? true : null"
```

Comment:
Consider updating these two (even though the button stuff is not strictly related to this) to correctly match the types in the Ionic docs. Updating button here ensures that setting null does not accidentally interfere with the anchor tag functionality in the future, if Ionic changes their implementation.

```suggestion
  [button]="_renderAsButton"
  [href]="_renderAsLink ? true : undefined"
```

You could also change the return type of _renderAsLink and simplify in template:   [href]="_renderAsLink"